packages/api/.env.example

```
@@ -28,3 +28,25 @@ ALLOW_OTP_FALLBACK=false
+# Stripe Payment Configuration
+STRIPE_SECRET_KEY=sk_test_your_stripe_secret_key_here
+STRIPE_PUBLISHABLE_KEY=pk_test_your_stripe_publishable_key_here
+STRIPE_WEBHOOK_SECRET=whsec_your_webhook_secret_here
+
+# Application URL (for payment redirect URLs)
+APP_URL=http://localhost:3000
+
+# Supabase Configuration (for file storage and future features)
+# Get from: https://supabase.com/dashboard/project/YOUR_PROJECT/settings/api
+SUPABASE_URL=https://your-project-id.supabase.co
+SUPABASE_KEY=your-supabase-anon-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)